src/som/vm/VmSettings.java

```diff
@@ -63,7 +63,7 @@
     DISABLE_TRACE_FILE = getBool("som.disableTraceFile", false) || (REPLAY && !KOMPOS_TRACING);
     TRACE_SMALL_IDS = getBool("som.smallIds", false);
 
-    ACTOR_TRACING = getBool("som.actorTracing", false);
+    UNIFORM_TRACING = getBool("som.actorTracing", false);
```

Comment:
Shouldn't the setting's name be turned into `som.uniformTracing` as well?

Reply:
yes, changing the flags etc would be a good idea. so from -at to -ut